File: app/src/main/java/net/mieczkowski/yelpbusinessexample/models/business/YelpBusinessWrapper.java
package net.mieczkowski.yelpbusinessexample.models.business;

import com.fasterxml.jackson.annotation.JsonProperty;

import java.util.ArrayList;

/**
 * Created by Josh Mieczkowski on 10/19/2017.
 */

public class YelpBusinessWrapper {

    @JsonProperty("businesses")
    private ArrayList<YelpBusiness> yelpBusinesses;

    public YelpBusinessWrapper() {
    }

    public ArrayList<YelpBusiness> getYelpBusinesses() {

        return yelpBusinesses;
    }

    public void setYelpBusinesses(ArrayList<YelpBusiness> yelpBusinesses) {
        this.yelpBusinesses = yelpBusinesses;
    }
}

File: dal/src/main/java/net/mieczkowski/dal/services/businessLookupService/models/YelpBusinessWrapper.kt
package net.mieczkowski.dal.services.businessLookupService.models

import com.fasterxml.jackson.annotation.JsonProperty
import java.util.*

/**
 * Created by Josh Mieczkowski on 9/11/2018.
 */
class YelpBusinessWrapper {

    @JsonProperty("businesses")
    lateinit var YelpBusinesss: ArrayList<YelpBusiness>
}